src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -519,4 +519,7 @@
   <data name="TuplesNeedAtLeastTwoElements" xml:space="preserve">
     <value>Tuples must have at least two elements.</value>
   </data>
+  <data name="TupleUnderlyingTypeMustBeTupleCompatible" xml:space="preserve">
+    <value>The underlying type for a tuple must have the shape of System.ValueTuple.</value>
+  </data>
```

Comment:
We should simply use term that language spec will use. For now, I'd use "must be tuple compatible"

---
In reply to: [62417391](https://github.com/dotnet/roslyn/pull/11160#discussion_r62417391) [](ancestors = 62417391)